python/PiFinder/catalogs.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import sqlite3
 import time
+from db.db import Database, ObjectsDatabase
```

Comment:
Is this a new file/module yet to be checked in?

Reply:
there were indeed some essential pieces missing :)